api/app/routers/pteams.py

```diff
@@ -801,6 +801,31 @@ def get_tickets_by_service_id_and_package_id_and_vuln_id(
     return ret
 
 
+@router.get("/{pteam_id}/tickets/{ticket_id}", response_model=schemas.TicketResponse)
+def get_ticket(
+    pteam_id: UUID,
+    ticket_id: UUID,
+    current_user: models.Account = Depends(get_current_user),
+    db: Session = Depends(get_db),
+):
+    """
+    Get a ticket.
+    """
+
+    if not (pteam := persistence.get_pteam_by_id(db, pteam_id)):
+        raise NO_SUCH_PTEAM
+    if not check_pteam_membership(pteam, current_user):
+        raise NOT_A_PTEAM_MEMBER
+    if not (ticket := persistence.get_ticket_by_id(db, ticket_id)):
+        raise NO_SUCH_TICKET
+
+    service = ticket.dependency.service
+    if str(service.pteam_id) != str(pteam_id):
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail="No such pteam")
```

Comment:
```suggestion
        raise NO_SUCH_TICKET
```
PTeamは存在するのでメッセージがおかしいです。
本来は「指定したPTeamの中に指定したTicketが無い」ですが、"No such ticket"で十分と思います。

Reply:
意図が伝わらなかったようですみません。
"No such ticket"であれば、Suggested changeで修正提案した通り、共通化してNO_SUCH_TICKETとすべきです。